client/client.go

```diff
@@ -388,19 +390,87 @@ func createClientWithKeyspace(
 	return c, nil
 }
 
-// NewClientWithKeyspaceName creates a client with context and the specified keyspace name.
-func NewClientWithKeyspaceName(
+// APIVersion is the API version the server and the client is using.
```

Comment:
```suggestion
// APIVersion is the API version used by the server and the client.
```

Reply:
They are almost the same.

Reply:
nope. The red one has typo. Alternative "APIVersion is the API version that the server and the client are using."